src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj

```diff
@@ -557,6 +557,9 @@
     <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.VirtualNetworkGatewayConnectionTests\TestVirtualNetworkGatewayConnectionwithIpsecPoliciesCRUD.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.VirtualNetworkGatewayConnectionTests\TestVirtualNetworkGatewayP2SAndSKU.json">
```

Comment:
The path is wrong here. It should be:
`SessionRecords\Commands.Network.Test.ScenarioTests.VirtualNetworkGatewayTests\TestVirtualNetworkGatewayP2SAndSKU.json`
The word `Connection` isn't in the folder path for this file.